docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '3.3'
```

Comment:
Eu não consegui correr isto inicialmente porque quando reestruturaste o projeto, esqueceste-te de incluir o `package.json` no `src`

Reply:
Verdade 😓 . Acabei de corrigir isso, não fiz re-build após atualizar o arquivo. Agora vai.

Reply:
Even after this fix I am not able to run this.
I am using the following command:
`sudo rm -r postgres/data && docker compose build --no-cache && docker-compose up`
The error reported is:
```
lovely_backend-node-1      |
lovely_backend-node-1      | > pg-promise-test@1.0.0 test /app
lovely_backend-node-1      | > node ./node_modules/ts-node/dist/bin.js src/index.ts
lovely_backend-node-1      |
lovely_backend-node-1      | internal/modules/cjs/loader.js:638
lovely_backend-node-1      |     throw err;
lovely_backend-node-1      |     ^
lovely_backend-node-1      |
lovely_backend-node-1      | Error: Cannot find module '/app/node_modules/ts-node/dist/bin.js'
lovely_backend-node-1      |     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
lovely_backend-node-1      |     at Function.Module._load (internal/modules/cjs/loader.js:562:25)
lovely_backend-node-1      |     at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)
lovely_backend-node-1      |     at startup (internal/bootstrap/node.js:283:19)
lovely_backend-node-1      |     at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)
lovely_backend-node-1      | npm ERR! code ELIFECYCLE
lovely_backend-node-1      | npm ERR! errno 1
lovely_backend-node-1      | npm ERR! pg-promise-test@1.0.0 test: `node ./node_modules/ts-node/dist/bin.js src/index.ts`
lovely_backend-node-1      | npm ERR! Exit status 1
lovely_backend-node-1      | npm ERR!
lovely_backend-node-1      | npm ERR! Failed at the pg-promise-test@1.0.0 test script.
lovely_backend-node-1      | npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
lovely_backend-node-1      | npm WARN Local package.json exists, but node_modules missing, did you mean to install?
lovely_backend-node-1      |
lovely_backend-node-1      | npm ERR! A complete log of this run can be found in:
lovely_backend-node-1      | npm ERR!     /root/.npm/_logs/2022-02-04T12_05_52_190Z-debug.log
```

Postgres seems to be running fine, but I'll only be able to be certain of that after the dash starts running queries on it.